instrumentation/opentelemetry-instrumentation-openai-v2/src/opentelemetry/instrumentation/openai_v2/utils.py

```diff
@@ -27,6 +28,18 @@
     server_attributes as ServerAttributes,
 )
 
+OTEL_INSTRUMENTATION_OPENAI_CAPTURE_MESSAGE_CONTENT = (
+    "OTEL_INSTRUMENTATION_OPENAI_CAPTURE_MESSAGE_CONTENT"
+)
+
+
+def is_content_enabled() -> bool:
+    capture_content = environ.get(
+        OTEL_INSTRUMENTATION_OPENAI_CAPTURE_MESSAGE_CONTENT, None
+    )
+
+    return bool(capture_content)
```

Comment:
NIt: Any truthy value will return true (any non-empty string would return true). Do we want that or do we want explicitly "true" string like we have for other [env vars](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-logging/src/opentelemetry/instrumentation/logging/__init__.py#L132). Not a strong opinion though.

Reply:
great catch, thanks!